intelmq/lib/bot.py

```diff
@@ -394,5 +399,56 @@ def __load_harmonization_configuration(self):
                             "Key %s is not valid." % _key)
 
 
+class ParserBot(Bot):
+
+    def parse_csv(self, report):
+        """
+        CSV parser.
+        """
+        raw_report = utils.base64_decode(report.get("raw"))
+        for line in csv.reader(io.StringIO(raw_report)):
+            yield line
+
+    def parse(self, report):
+        """
+        Default parser yielding stripped lines.
+        """
+        for line in utils.base64_decode(report.get("raw")).splitlines():
+            yield line.strip()
+
+    def process(self):
+        self.tempdata = []
```

Comment:
Maybe this needs a little comment at the front explaining the meaning and use of it?